storage/scylladb_identity.h

```diff
@@ -16,6 +16,9 @@ extern "C" {
 
 /**
  * @file storage/scylladb_identity.h
+ * @brief Implementation of identity table and corresponding insertion and selection functions.
+ *
+ * Data stored in the identity table would be used for transaction reattachment.
```

Comment:
Improve the descriptions. You shall explain the facilities about identity table.